### ISSUE TITLE
[Bug] 输出错位且分散

### ISSUE THREAD

**Issue by bytefluxx** [huang229/auto_tooth_arrangement#24]
您好，

我尝试在我的数据集上运行该项目，但结果似乎 错位且分散。具体情况如下：

输出结果未达到预期的对齐效果。

元素（牙齿 / 点 / 对象）出现 重叠或位置错误。

**Comment by yeyuxmf** [comment]
@bytefluxx 这个开开源模型，是有一组数据的，你直接跑main.py就能直接训练，而且能看到训练收敛，你测试这组数据也能得到很好的结果，说明模型是没有问题的。
但是你训练你的数据没达不到预期效果，大概率训练时损失值也没有像readme里面展示那样收敛，我猜测是你的数据预处理有问题，数据加载的脚本注释了很多可视化代码，你完全可以用来检查数据预处理的正确性。
还有一个很重要的问题，就是你有没有深度学习基础，要是没有，你要对模型就太难了。
如果你有深度学习基础，可以留下微信号，我加你沟通。

**Comment by bytefluxx** [comment]
@yeyuxmf 您好，
非常感谢您的回复和详细的解答！我在运行自己的数据集时，确实发现训练的损失值没有像 README 中展示的那样收敛，输出结果也存在错位和分散的问题。您提到数据预处理可能存在问题，我非常认同这个观点。我已经尝试查看数据加载脚本中的可视化代码，但对于如何正确进行数据预处理，尤其是数据规范化（统一尺度、坐标系对齐、中心化），我还有一些疑问。
具体来说，我想请教以下几点：

在数据预处理过程中，如何确保数据集的尺度统一？比如，是否需要对输入数据进行某种缩放或标准化操作？
对于坐标系的对齐，您是否有推荐的方法或脚本示例来 garantir 所有数据的参考系一致？
关于数据的中心化，是否有特定的步骤或工具可以用来检查和调整数据的中心位置？

我有一定的深度学习基础，但在数据预处理的细节上可能需要更多指导。如果方便的话，能否在 GitHub 上分享一些关于上述规范化步骤的具体建议或代码示例？我会非常感谢您的进一步帮助！
期待您的回复，谢谢！

**Comment by yeyuxmf** [comment]
@bytefluxx 
统一尺度和单颗牙齿去中心化：arangementnet.py：

class teeth_arangement_model(nn.Module):
    def __init__(self, in_channel=3):
        super(teeth_arangement_model, self).__init__()

        self.local_fea = mae_vit_base_patch16()

    def forward(self, teeth_points, centerp):

        B,T,N,C = teeth_points.shape
        teeth_points = (teeth_points - centerp).permute(0, 1, 3, 2)#teeth_points.permute(0, 1, 3, 2)#
        cenv = centerp.permute(0, 1, 3, 2).expand(-1,-1,-1, N)

        tmaxv = torch.max(torch.abs(teeth_points.reshape(B, T, N*C)), dim=-1)[0].view(B, T,1, 1)
        teeth_points = teeth_points / tmaxv  #You can remove this code.
        teeth_points = torch.cat([teeth_points, cenv], dim=-2)

坐标系对齐：
   所有的正畸前的数据坐标系是一致的就可以了,只要厂商是一家，那么它采集的数据坐标系都是一致的。

整体去中心化load_train_data.py：
   那就是正畸前的数据不单坐标系一致，它们的中心点都是(0, 0, 0)
   Rteeth_points = Rteeth_points.reshape(cfg.teeth_nums * cfg.sam_points, cfg.dim)
   rcpoint = np.mean(Rteeth_points, axis=0)
   Rteeth_points = Rteeth_points - rcpoint
   Rteeth_points = Rteeth_points.reshape(cfg.teeth_nums, cfg.sam_points, cfg.dim)

    Gteeth_points = Gteeth_points.reshape(cfg.teeth_nums * cfg.sam_points, cfg.dim)
    Gteeth_points = Gteeth_points - np.mean(Gteeth_points, axis=0)
    Gteeth_points = Gteeth_points.reshape(cfg.teeth_nums, cfg.sam_points, cfg.dim)
